dae/dae/gpf_instance/gpf_instance.py

```diff
@@ -86,8 +81,12 @@ def __init__(
 
     @property  # type: ignore
     @cached
-    def reference_genome(self) -> ReferenceGenome:
-        """Returns reference genome defined in the GPFInstance config"""
+    def reference_genome(self):
+        """Return reference genome defined in the GPFInstance config."""
+        # pylint: disable=import-outside-toplevel
+        from dae.genomic_resources.reference_genome import \
```

Comment:
I feel like we discussed this but I may have forgotten, why do we use inline imports in these cases?